Keas.Mvc/ClientApp/components/Spaces/SpacesDetailsEquipment.tsx

```diff
@@ -33,32 +34,32 @@ export default class SpacesDetailsEquipment extends React.Component<IProps, ISta
 
     public async componentDidMount() {
         this.setState({ loading: true });
-        const equipment = !this.props.roomKey ? [] :
-            await this.context.fetch(`/api/${this.context.team.name}/equipment/getEquipmentInSpace?roomKey=${this.props.roomKey}`);
+        const equipment = !!this.props.spaceId === null ? [] :
```

Comment:
@laholstege not sure what you are trying to do here... I don't think you need the !! maybe.  `this.props.spaceId === null` evaluates to a boolean so !! is doing nothing for you.  Maybe you meant `!!this.props.spaceId`?